src/client/App.jsx

```diff
@@ -1,11 +1,11 @@
 import './App.css';
 
-import Registration from './components/Registration';
+import Login from './components/Login';
 
 export default function App() {
   return (
     <div className="App">
-      <Registration />
+      <Login />
```

Comment:
Just to be clear, we are only importing/removing imports like this for testing before routing is implemented?

Reply:
Yes, that was my plan. I just need to be able to see what I'm doing is all